sdk/spring/spring-cloud-azure-appconfiguration-config-web/src/main/java/com/azure/spring/cloud/config/web/implementation/pushbusrefresh/AppConfigurationBusRefreshEventListener.java

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.azure.spring.cloud.config.web.pushbusrefresh;
+package com.azure.spring.cloud.config.web.implementation.pushbusrefresh;
```

Comment:
Line 15, if this is already defined as a bean, then there's no need to add the `@Component` annotation.